### DISCUSSION TITLE
[BUG] compilation

### DISCUSSION THREAD

**Discussion by Anthony3364** [Estia-1a/projetGenieInfo_public#84] [Project Installation and Configuration]
Bonjour, 

J'ai ce problème quad je compilation lorsque je lance un built. 
Pouvez-vous m'aider ? 

Merci 
![image](https://user-images.githubusercontent.com/104560562/173572070-eeb2ac93-3330-44f7-89e1-240edbff1f2c.png)

**Comment by adisdier** [comment]
@Anthony3364 You called one of your folders "Système gestion version" but gcc obviously cannot manage accents (look more closely at the output you gave) => Try removing any accent and space from your project path, and then try compiling again.

**Comment by Anthony3364** [comment]
Effectivement j'ai changé de dossier maintenant, ca fonctionne. 
MERCI :)
